docs/FRSKY_R9.md

```diff
@@ -21,7 +21,7 @@ The Frsky R9M, R9M Lite Pro transmitter modules and R9 MX, R9 MM, R9 Mini receiv
   </tbody>
 </table>
 
-Note that R9 hardware cannot connect with other mLRS boards which support 868/915 MHz. This is because it uses the SX127x LoRa chipset, which is incompatible with the newer SX126x LoRa chipset of the other boards when the SF6 spreading factor  used by the mLRS 19 Hz mode is selected. The older SX127x chipset also does not support SF5 which is required for the mLRS 31 Hz mode.
+Note that R9 hardware cannot connect with other mLRS boards which support 868/915 MHz. This is because it uses the SX127x LoRa chipset, which is incompatible with the newer SX126x LoRa chipset of the other boards when the SF6 spreading factor used by the mLRS 19 Hz mode is selected. The older SX127x chipset also does not support SF5 which is required for the mLRS 31 Hz mode.
```

Comment:
Small rewrite:

Note that R9 hardware cannot connect with mLRS boards which support 868/915 MHz and use the SX126x LoRa chipset. This is because R9 hardware uses the SX127x LoRa chipset, which is incompatible with the SF6 spreading factor used by the mLRS 19 Hz mode. In addition, the SX127x does not support the SF5 spreading factor which is used for the mLRS 31 Hz mode.

Reply:
NICE. done.